src/classes/Countertop.js

```diff
@@ -9,20 +10,36 @@ import CountertopTopology from './CountertopTopology'
  * the countertop architecture.
  */
 class Countertop {
+	logger = null
+
 	stations = []
 
+	/**
+	 * @param  {Logger} options.logger A logger with methods for all TV Kitchen logLevels.
+	 */
+	constructor({
+		logger = consoleLogger,
+	} = {}) {
+		this.logger = logger
+	}
+
 	/**
 	 * Register a new, configured appliance into the countertop.
 	 * Appliances that ingest Payload streams may be cloned as new streams are created.
 	 *
-	 * @param  {Class}  Appliance     The class of the IAppliance being registered.
-	 * @param  {String} configuration The configuration used to register the appliance station.
-	 * @return {CountertopStation}    The station created for this appliance.
+	 * @param  {Class}  Appliance                 The class of the IAppliance being registered.
+	 * @param  {String} options.applianceSettings The settings to be passed to the appliance.
+	 * @return {CountertopStation}                The station created for this appliance.
 	 */
-	addAppliance = (Appliance, configuration) => {
+	addAppliance = (Appliance, {
+		applianceSettings = {},
```

Comment:
This might feel like overkill to use destructuring for.  The thinking is that there are going to be additional settings that are more specific to countertop topologies (e.g. "ignoreStations" etc.)

Reply:
It feels a little aesthetically weird to imagine calling `addAppliance` like:
```js
addAppliance(appliance, { applianceSettings: { setting: false } })
```

But not _that_ weird.

What do you mean by "overkill?"

Reply:
I just meant that without other parameters being destructured it seems silly to destructure parameters.

That being said... you're absolutely right.  I'm going to take this back.  Maybe we should just have three parameters:

1 -> the appliance
2 -> appliance settings
3 -> whatever relevant countertop settings we end up deciding on.

Where 3 is the one that might be destructured once we decide on those settings.

Reply:
I just meant that without other parameters being destructured it seems silly to destructure parameters.

That being said... you're absolutely right. I'm going to take this back. Maybe we should just have three parameters:

1 -> the appliance
2 -> appliance settings
3 -> whatever relevant countertop settings we end up deciding on.

Where 3 is the one that might be destructured once we decide on those settings.